updateList.php

```diff
@@ -0,0 +1,39 @@
+<?php
+session_start();
+
+if(isset($_GET['id'])){
+    $hobbieName = $_GET['id'];
+
+if($hobbieName === "Dancing"){$hobbieID = 1;}
+if($hobbieName === "Listening Music"){$hobbieID = 2;}
+if($hobbieName === "Cricket"){$hobbieID = 3;}
+if($hobbieName === "Singing"){$hobbieID = 4;}
+if($hobbieName === "Fooseball"){$hobbieID = 5;}
+if($hobbieName === "Reading Novels"){$hobbieID = 6;}
```

Comment:
hobbies ki array bana le and then just use the array_search() function go get the index of hobbieName instead of writing 5 lines. You can also use for exception handling when user sends a hobbie that isn't present in the array

Reply:
Yeah! Updated the code my deleting the above and making an array of hobbies.